src/Storages/System/StorageSystemBuildOptions.generated.cpp.in

```diff
@@ -8,18 +8,16 @@ const char * auto_config_build[]
     "SYSTEM", "@CMAKE_SYSTEM_NAME@",
     "VERSION_GITHASH", "@VERSION_GITHASH@",
     "VERSION_REVISION", "@VERSION_REVISION@",
-    "VERSION_DATE", "@VERSION_DATE@",
```

Comment:
@azat lets remove this in separate pull request.

Reply:
I am almost sure is used somewhere, better to revert this change. This pull request need to be merged, because of minor issue.

Reply:
FWIW there is `GIT_DATE` that is something similar, it is date time, while `VERSION_DATE` is date only type.